mesonbuild/interpreter.py

```diff
@@ -980,8 +981,11 @@ def first_supported_argument_method(self, args, kwargs):
         mlog.log('First supported argument:', mlog.red('None'))
         return []
 
-class ModuleState:
-    pass
+ModuleState = namedtuple('ModuleState', [
```

Comment:
I asked because I was confused what that change had to do with cleaning up files properly. It doesn't make any difference to have it though, so it's up to Jussi :)